File: kotak_kotak.java
public class kotak_kotak{
	public static void main(String[]args){
		int i;
		int j;
		int tinggi = 3;
		// Proof of Concept
		// 0 2 2
		// 2 0 2
		// 2 2 0
		// Jika i = i dan j = i maka akan cetak angka 0
		// selain itu angka 2
		// j > tinggi print new line
		for (i=0;i<tinggi;i++){
			for (j=0;j<tinggi;j++){
				if (i == i && j==i){
					//jika i = i print 0
					//jika j = i print 0
					System.out.print(" 0 ");
				}else{
					System.out.print(" 2 ");
				}
			}
			System.out.println("");
		}
	}
}

File: Python/kotak_kotak.py
# 1 2 2
# 2 1 2
# 2 2 1
# Jika perulangan pertama adalah angka 1 maka cetak angka 1
# 
tinggi = 10
for i in range(0,tinggi,1):
	for j in range(0,tinggi,1):
		#if i == 0 or i == tinggi - 1 or j == 0 or j == tinggi - 1:
			#print ('*',end='')
		#else:
			#print (end=' ')
		##print ('*',end=' ')
		if i == i and j==i:
			print ('  ',end='')
		else:
			print (' 2 ',end='')
	print (' ')